### DISCUSSION TITLE
Migration hooks

### DISCUSSION THREAD

**Discussion by doender** [prisma/prisma#15960] [Q&A]
I'm looking for a way to modify existing data before/after a migration has ran, e.g. after a migration from separate first and last name columns to a full name column. Ideally I would like to use the prisma client to modify this data instead of raw SQL. Is this possible?

**Comment by austincrim** [comment]
Hey @doender 👋,

Please take a look at [this existing feature request](https://github.com/prisma/prisma/issues/15384) and add any additional thoughts/use cases.

**Reply by anitacung** [reply]
👋 Hi @doender,

I'm going through pending GitHub Discussions to check if you need any help. Have you looked into Austin’s suggestion? Feel free to let us know if you need additional help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
